studies/tests.py

```diff
@@ -35,9 +35,9 @@
 
 You and your child can participate any time you want by going to "The Most Fake Study Ever" on Lookit ({base_url}/studies/{study_uuid}/). If you have any questions, please reply to this email to reach the ECCL at faker@fakelab.com.
```

Comment:
We still have 'on Lookit' in this line and above (line 24).
I'm assuming that this text is matched against the email template text file, rather than html file, which is why the test passed, and that the test will fail if you update the email template txt.